### ISSUE TITLE
Release v1.10

### ISSUE THREAD

**Issue by dondi** [kdahlquist/GRNmap#382]
If possible, wrap-up release for the semester's work. Procedure can be found here: https://github.com/kdahlquist/GRNmap/wiki/How-To-Create-a-New-Release

**Comment by jtorre39** [comment]
I've noticed on the instructions after the first step of merging beta into master, the next step is to tag the commit for the release. But the instructions do not mention pushing the commit after merging into master. If the tests are passing, do I have the go ahead to push the merging of beta into master onto the repo?

Otherwise I am unable to tag the release to my knowledge. Any thoughts @dondi ?

**Comment by jtorre39** [comment]
Additionally, I was wondering what version number this would be. Our most recent on is v1.8. Are the changes sufficient to call it v2.0? I'll also defer to @kdahlquist on these matters.

**Comment by kdahlquist** [comment]
Need advice from @dondi on how to do the merging with forks.  @jtorre39 has merged beta into master on his fork, but should he now push to the master branch on the main repository and then tag?

The confusion is that the instructions on the wiki are pre-dating the use of forks to manage the code.

**Comment by jtorre39** [comment]
I figured it out! The commits are tagged and the release is drafted! I'm testing the binaries I uploaded right now to make sure it was packaged properly.

**Comment by jtorre39** [comment]
I ran a test sheet and compared it to the results from the code on our master branch and it checks out!

**Comment by kdahlquist** [comment]
Let's pull the trigger on this!

**Comment by jtorre39** [comment]
Released!

**Comment by jtorre39** [comment]
Website updated!

**Comment by kdahlquist** [comment]
Updated title to this issue for the upcoming release of v1.10.  @dondi and @kdahlquist will do it in the room together to troubleshoot getting Zenodo to work.

**Comment by kdahlquist** [comment]
So it turns out that Zenodo does have a badge for us. See: https://zenodo.org/record/1206314#.W8ZNf_lRfmg I've added it to the ReadMe at the top level of the repository. So I guess the last time I checked there was just a lag and I didn't see it yet.  We can move forward with the release on Thursday.  I'm going to work on the ReadMe and sample files #407 now.  We'll close this issue when we do the release on Thursday.
